tests/clubhouse/test_plugin.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import absolute_import
+
+import responses
+
+from exam import fixture
+from django.contrib.auth.models import AnonymousUser
+from django.test import RequestFactory
+from sentry.testutils import PluginTestCase
+from sentry.utils import json
+
+from sentry_plugins.clubhouse.plugin import ClubhousePlugin
+
+
+class ClubhousePluginTest(PluginTestCase):
+    @fixture
+    def plugin(self):
+        return ClubhousePlugin()
+
+    @fixture
+    def request(self):
+        return RequestFactory()
+
+    def test_conf_key(self):
+        assert self.plugin.conf_key == 'clubhouse'
+
+    def test_entry_point(self):
+        self.assertAppInstalled('clubhouse', 'sentry_plugins.clubhouse')
+        self.assertPluginInstalled('clubhouse', self.plugin)
+
+    def test_get_issue_label(self):
+        group = self.create_group(message='Hello world', culprit='foo.bar')
+        assert self.plugin.get_issue_label(group, {
+            'id': 1,
+        }) == 'Clubhouse Story #1'
+
+    @responses.activate
+    def test_get_issue_url(self):
+        responses.add(
+            responses.GET,
+            'https://api.clubhouse.io/api/v2/stories/1',
+            json={'id': 1, 'app_url': 'https://app.clubhouse.io/example-org/story/1'}
+        )
+
+        self.plugin.set_option('repo', 'getsentry/sentry', self.project)
+        group = self.create_group(message='Hello world', culprit='foo.bar')
+        assert self.plugin.get_issue_url(group, {
+            'id': 1,
+            'url': 'https://app.clubhouse.io/example-org/story/1',
+        },
+        ) == 'https://app.clubhouse.io/example-org/story/1'
+
+    def test_is_configured(self):
+        assert self.plugin.is_configured(None, self.project) is False
+        self.plugin.set_option('token', '12345678-1234-1234-1234-1234567890AB', self.project)
+        assert self.plugin.is_configured(None, self.project) is False
+        self.plugin.set_option('project', '1234', self.project)
+        assert self.plugin.is_configured(None, self.project) is True
+
+    def test_validate_config(self):
```

Comment:
not sure theres much to help here -- we definitely need to spend more time making these things easier to test